Rise-Vision/common-header
# Fix Weather style

## Description
Changes to the Weather attributes editor as per [this doc](https://docs.google.com/document/d/13hSHSo8FMMo9CXDo4lutGRXcLCwCTGefOC-EXPTMhog/edit?ts=5d374725#bookmark=id.3kfvkstwwhjj) to match the [styleguide](https://docs.google.com/document/d/1JaurbTMAWsHZIauyrENMVdM1shwdzb5BqtKkjSURVoI/edit#bookmark=id.ye661i2potlm).

## Motivation and Context
Updated the UI to match the recent changes to the styleguide.

## How Has This Been Tested?
Tested the stage link below on supported browsers, confirmed no impact on other elements.

Staged at: https://apps-stage-1.risevision.com/templates/edit/2f490f69-7e31-4108-947e-41b9a9d0a828/?cid=554d3ef9-78b7-4726-8f46-c1ec140c166d

## Release Plan:
- As the Submitter, upon requesting review of this pull request, I confirm that the [Release Checklist](https://help.risevision.com/hc/en-us/articles/360031119991) has been completed. 

**Manual Test**  Tested on supported browsers
**Automated Test** No need to update, as it was only a style change
**Monitoring** Will validate the release once in Production
**Rollback** Rollback can be done by reverting the PR and following the deployment process
**Documentation** Minor style changes, no need to update documentation
**Internal Communication** Minor style changes, no need for internal communication

- As the Reviewer, upon approving the changes in this PR, I confirm I have reviewed and I agree that the [Release Checklist](https://help.risevision.com/hc/en-us/articles/360031119991) has been completed

#### Release Checklist Items Skipped?
If any Release Checklist items were intentionally skipped, please provide which ones and the reasons why
